synfig-studio/src/synfigapp/vectorizer/global_variables.cpp

```diff
@@ -0,0 +1,37 @@
+/* === S Y N F I G ========================================================= */
+/*!	\file action_param.cpp
```

Comment:
you could update the file name information and some other information in the comments here